<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleHostingDialog</class>
 <widget class="QDialog" name="BattleHostingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Description:</string>
       </property>
       <property name="buddy">
        <cstring>descriptionLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Password:</string>
       </property>
       <property name="buddy">
        <cstring>passwordLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>P&amp;ort:</string>
       </property>
       <property name="buddy">
        <cstring>portSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="portSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>8452</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Ma&amp;x. Players:</string>
       </property>
       <property name="buddy">
        <cstring>maxPlayersSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="maxPlayersSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>4</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>&amp;Rank:</string>
       </property>
       <property name="buddy">
        <cstring>rankComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="rankComboBox">
       <item>
        <property name="text">
         <string>Newbie</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/rank0.xpm</normaloff>:/icons/rank0.xpm</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beginner</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/rank1.xpm</normaloff>:/icons/rank1.xpm</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Average</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/rank2.xpm</normaloff>:/icons/rank2.xpm</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Above Average</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/rank3.xpm</normaloff>:/icons/rank3.xpm</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Experienced</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/rank4.xpm</normaloff>:/icons/rank4.xpm</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Highly Experienced</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/rank5.xpm</normaloff>:/icons/rank5.xpm</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Veteran</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/rank6.xpm</normaloff>:/icons/rank6.xpm</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Mod:</string>
       </property>
       <property name="buddy">
        <cstring>modComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="modComboBox"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>M&amp;ap:</string>
       </property>
       <property name="buddy">
        <cstring>mapComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="mapComboBox"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BattleHostingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>245</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
